packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -33,6 +33,12 @@ export abstract class CollectorExporterBrowserBase<
   ExportItem,
   ServiceRequest
 > {
+  private DEFAULT_HEADERS = {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  };
+  private DEFAULT_BLOB_TYPE = { type: 'application/json' };
```

Comment:
imho it should not be a part of CollectorExporterBrowserBase.ts as it is only used if beacon is used -> move to util

Reply:
the logic deciding if beacon is used , is a decision that `CollectorExporterBase` already has made before sendWithBeacon util function.